### ISSUE TITLE
Configurable Screen Darkening

### ISSUE THREAD

**Issue by LuminaSapphira** [CerulanLumina/HealthHungerTweaks#6]
While playing Invasion a bit, it occurs to me I want the screen darkening to be adjustable. The default is alright for default config, but in invasion with healing disabled, the red can get annoying after a while.

Goals:
 - Change darkening start.
 - Change proportion curve
    - (Maybe with equation?? that'd be cool)

**Comment by Lodugh** [comment]
Yeah, I just wanted to open an Issue about that by myself. Would be great!

**Comment by LuminaSapphira** [comment]
Alright so the equations won't happen, but I'll allow people to change when it starts in the config, and maybe craft a reusable item to remove the darkening temporarily in game.

**Comment by Darkmega18** [comment]
Is this implemented or in the works? I've noticed as health grows with mods like scaling health the red screen darkening becomes earlier and earlier rather than being at an exact value of lacking when it starts appearing or at a percentage of health, so I've had to turn it off otherwise you could still be at half health when you've got 20 hearts and the screens already fairly dark when you're not super in trouble yet. since it feels like it runs off health lost rather than health remaining or something.

**Comment by LuminaSapphira** [comment]
This will be in the next major release (ie not a simple bugfix update). Sorry I haven't been working on this very often, I've been a bit busy.

**Comment by Darkmega18** [comment]
Thats alright, stuff happens. just checking it out it out and reporting my findings since the issues page exists. since it is good to have the screen darkening so I remember to check my health, but when the bar grows, it just becomes a false alarm. :<
